### PR TITLE
Change 3.0 url to 4.0

### PR DESCRIPTION
**Pull request by njaffer**
This is to fix a bug where it shows CC 4.0 but actually links to 3.0

### CODE REVIEW

**Comment by grosscol** [comment]
This will have to handle cases were the work was licensed under 3.0.  Can't have 3.0 links broken nor pointing to 4.0

**Comment by njaffer** [comment]
It has CC 3.0 in the rights authorities but  only displays 4.0 in UI

File: app/views/records/edit_fields/_rights.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         end ) + 
         ( content_tag :aside do
             t_uri(:description, scope: [ :rights, b.value ])
-        end )
+        end ) if !b.text.include?('3.0')
```
Comment (grosscol) [inline app/views/records/edit_fields/_rights.html.erb]: Is there way to do this from the RightsService such that it only returns currently valid options from `select_options` ?

Comment (njaffer) [inline app/views/records/edit_fields/_rights.html.erb]: @grosscol: Cannot find a simple way to do it. I thought this is a pretty clean and straightforward solution. I will need to update the tests though!

 If you have some ideas, do let me know.

File: spec/records/edit_fields/_rights.html.erb_spec.rb [hunk 2]
```diff
@@ -25,15 +25,10 @@
...
-  it "rights should have asides" do
-    num_asides = RightsService.select_options.length
-    expect(rendered).to have_selector(".generic_work_rights .radio aside", count: num_asides)
-  end
-
   it "rights should have descriptions for each right" do
     num_asides = RightsService.select_options.length
     RightsService.select_options.each do |right|
-      expect(rendered).to have_content(t_uri(:description, scope: [ :rights, right[1] ]))
+      expect(rendered).to have_content(t_uri(:description, scope: [ :rights, right[1] ]))if !right[1].include?('3.0')
```
Comment (grosscol) [inline spec/records/edit_fields/_rights.html.erb_spec.rb]: This is one of the reasons to make the change in the RightsService.  When we add rights 4.1, 4.2, ect.. we'd have to update the logic in the ui that uses it and add the logic into the tests.  This conditional will eventually end up as a compound one and finally a regex before it becomes too difficult to update.

**Pull request closed without merging**